apps/traincascade/cascadeclassifier.cpp

```diff
@@ -450,6 +452,7 @@ void CvCascadeClassifier::save( const string filename, bool baseFormat )
         CvSeq* weak;
         if ( cascadeParams.featureType != CvFeatureParams::HAAR )
             CV_Error( CV_StsBadFunc, "old file format is used for Haar-like features only");
+        fs << "{:" << ICV_HAAR_TYPE_ID;
```

Comment:
That makes sense I was wondering about that as well since one of my earlier tests did't work as expected but then they all started to. I can make the change.